config.lua

```diff
@@ -10,25 +10,32 @@ Config.Locations = {
     jobs = {
         ["electrician"] = {
             [1] = {
-                coords = vector4(1761.46, 2540.41, 45.56, 272.249)
+                coords = vector4(1761.46, 2540.41, 45.56, 272.249),
+                done = false -- Check to see if the job is done, leave this false
```

Comment:
I'm actually curious about: why do you put `done = false`?
Can't you do it in the script startup so people don't have to put it there? ^^

Reply:
I added it to make sure no one can do the same one all over each time

Reply:
And this way was easier